### PR TITLE
correction in PART/MAT review of Starter w/ S20

### PR DESCRIPTION
**Pull request by qiangRadioss**
#### Checklist

- [x] The title of the PR is reviewed
- [x] There is no text before the checklist section
- [x] The following two sections are filled (or deleted)
- [x] This PR has been previewed 

#### Description of the feature or the bug

wrong Isolid value of /BRICK20 (solid 20 node element) in Part/Mat Review of Starter outfile when the same Pid is shared by different solid parts

#### Description of the changes

Correction done on the Isolid of S20, as well for tetra4,tetra10 and penta6.

### CODE REVIEW

File: starter/source/elements/initia/initia.F [hunk 1]
```diff
@@ -3503,6 +3513,14 @@ SUBROUTINE OUTPART5(GROUP_PARAM_TAB,IPART,IPARG,IGEO, GEO ,PM )
 		     IF (IPARG(28,NG0)==8) NG = NG0
            END IF
           END IF		   
+           SELECT CASE (ISOLNOD)
+             CASE(4,10)
+		       IHBE=1
```
Comment (sebastienVilleneuve) [inline starter/source/elements/initia/initia.F:3518]: here some tabulation to remove

File: starter/source/elements/initia/initia.F [hunk 2]
```diff
@@ -3503,6 +3513,14 @@ SUBROUTINE OUTPART5(GROUP_PARAM_TAB,IPART,IPARG,IGEO, GEO ,PM )
...
+             CASE(6)
+		       IHBE=15
```
Comment (sebastienVilleneuve) [inline starter/source/elements/initia/initia.F:3520]: here some tabulation to remove

File: starter/source/elements/initia/initia.F [hunk 3]
```diff
@@ -3503,6 +3513,14 @@ SUBROUTINE OUTPART5(GROUP_PARAM_TAB,IPART,IPARG,IGEO, GEO ,PM )
...
+             CASE(20)
+		       IHBE=16
```
Comment (sebastienVilleneuve) [inline starter/source/elements/initia/initia.F:3522]: here some tabulation to remove

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
@qiangRadioss 
some changes to perform ( tabulations to remove)

**Comment by qiangRadioss** [comment]
tabulations removed

**Review by sebastienVilleneuve — APPROVED** [review]

**Comment by sebastienVilleneuve** [comment]
Thanks @qiangRadioss

**Pull request merged**